docs/UserGuide.md

```diff
@@ -573,44 +597,46 @@ Checks if a group fulfils the group requirements of the course.
 
 #### Expected Output(Success):
 If the group fulfils the group requirements,
-* Message: "Group fulfils the diversity requirements of CS2103T. Group GROUP_NUMBER"
+* Message: "Group GROUP_NUMBER
+
```

Comment:
is this a extra blank line?

Reply:
no, it isn't an extra blank line but i've added breaks to improve readability

Reply:
can use <br> instead or it isnt possible?